PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+## Issue
+
+Link any related issue(s) in this section.
+
+## Intent
+
+What this PR intends to achieve.
+
+## Implementation
+
+What code changes have been made to achieve the intent.
+
+## Checks
+
+- [ ] Code is formatted correctly (`npm run lint:fix`).
+- [ ] Any new functionality has been unit tested.
+- [ ] All unit tests are passing (`npm test`).
+- [ ] All CI checks are green.
```

Comment:
Could we add:
 [] JSDoc Comments have been added or updated

Reply:
Done!